eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -212,6 +212,9 @@ stages:
               pool:
                 name: azsdk-pool-mms-ubuntu-2004-general
                 vmImage: MMSUbuntu20.04
+
+              variables:
+                DocValidationImageId: azuresdkimages.azurecr.io/javarefautocr:latest
```

Comment:
Is there a way we can pin this image version easily instead of using latest?

Reply:
We are still in the process of testing the docker image. Once it is stable we can pin the specific version.